dags/resources/stages/parse/table_definitions/aragon/ACL_event_ChangePermissionManager.json

```diff
@@ -0,0 +1,51 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "name": "app",
+                    "type": "address"
+                },
+                {
+                    "indexed": true,
+                    "name": "role",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": true,
+                    "name": "manager",
+                    "type": "address"
+                }
+            ],
+            "name": "ChangePermissionManager",
+            "type": "event"
+        },
+        "contract_address": "SELECT proxy FROM ref('Kernel_event_NewAppProxy') WHERE appId = '0xe3262375f45a6e2026b7e7b18c2b807434f2508fe1a2a3dfb493c7df8f4aad6a'",
```

Comment:
Using this pattern for all of these parsers. Basically the `appId` decides what kind of Aragon app (Voting, Finance, etc) it is.